hyperon_das_atomdb/database.py

```diff
@@ -78,6 +88,24 @@ def _add_node(self, node_params: Dict[str, Any]) -> Dict[str, Any]:
         return (handle, node)
 
     def _add_link(self, link_params: Dict[str, Any], toplevel: bool = True) -> Dict[str, Any]:
+        reserved_parameters = [
+            '_id',
+            'composite_type_hash',
+            'is_toplevel',
+            'composite_type',
+            'named_type',
+            'named_type_hash',
+            'key_n',
+        ]
+
+        if any(
+            item in reserved_parameters or item.startswith('key') for item in link_params.keys()
```

Comment:
This doesn't fix the error I pointed out.

Any user key like "keyword" would trig this exception. You can merge this way but I added another issue to fix this: https://github.com/singnet/das-atom-db/issues/57

Reply:
I change for this: `re.match(r'key_\d+', item)`. I think it's better

Reply:
Yes but you should compile the regular expression once first and use here only the compiled version of it. This is a performance improvement because regexp compilation is expensive.

Reply:
Fixed. I put it as a static variable